src/main/java/io/jenkins/plugins/pipelinegraphview/multipipelinegraphview/PipelineRun.java

```diff
@@ -1,15 +1,34 @@
 package io.jenkins.plugins.pipelinegraphview.multipipelinegraphview;
 
+import hudson.Util;
+import hudson.model.Result;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 
 public class PipelineRun {
 
-    private String id;
-    private String displayName;
+    private final String id;
+    private final String displayName;
+    private final String duration;
+    private final String result;
+    private final String startTime;
 
     public PipelineRun(WorkflowRun run) {
         this.id = run.getId();
         this.displayName = run.getDisplayName();
+        this.duration = Util.getTimeSpanString(run.getDuration());
+
+        Result buildResult = run.getResult();
+        this.result = (buildResult != null) ? buildResult.toString() : "IN_PROGRESS";
+
+        SimpleDateFormat sdf = new SimpleDateFormat("d MMM, hh:mm", Locale.getDefault());
+        // TODO support user timezones from UserProperty
+        sdf.setTimeZone(TimeZone.getDefault());
```

Comment:
Users are able to set their own timezone in their user configuration, it would be good to support this, and also the Jenkins timezone property won't be supported by this core Java feature.